knora-ontologies/knora-base.ttl

```diff
@@ -561,6 +561,7 @@
     rdfs:comment            "Indicates that this resource is part of another resource"@en ;
     :subjectClassConstraint :Resource ;
     :objectClassConstraint  :Resource ;
+    :isEditable             true ;
```

Comment:
Hm, I think I don't understand why here everything is set to `:isEditable` `true`. Shouldn't `knora-base` things not be editable?